etl/recipes/recipe_unpop-3.yaml

```diff
@@ -182,6 +189,8 @@ cooking:
                   "*": sum
           result: unpop-datapoints-3-2-after2015
 
+        # merge estimates and medium for one long time-series
+        # ? why not just filter on estimate and medium?
```

Comment:
because estimate and medium both have datapoints for 2015, if just filter on estimate and medium and remove the variant dimension, there will be 2 2015 datapoints for each country

Reply:
Okay, by doing it this way it's year-agnostic. Instead of explicitly filtering out 2015-medium rows, we always overwrite medium by estimate.